### ISSUE TITLE
I would like to take over this repository.

### ISSUE THREAD

**Issue by gusbemacbe** [spiffin/spf_codemirror#4]
Hello @spiffin !

I am very sorry for your health, I do not know if you are still alive. 
I am advanced in PHP since 2015. I will give PHP classes to students at my college.
If you accept to transfer your repositories to me, contact me.

My e-mail is gusbemacbe@gmail.com.

Gustavo Costa

**Comment by spiffin** [comment]
Hello Gustavo

Thanks for your message. Do you have much experience of using Textpattern?

Simon




On 1 March 2018 at 01:02:16, Gustavo Reis (notifications@github.com) wrote:

Hello @spiffin <https://github.com/spiffin> !

I am very sorry for your health, I do not know if you are still alive.
I am advanced in PHP since 2015. I will give PHP classes to students at my
college.
If you accept to transfer your repositories to me, contact me.

My e-mail is gusbemacbe@gmail.com.

Gustavo Costa

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/spiffin/spf_codemirror/issues/4>, or mute the thread

**Comment by gusbemacbe** [comment]
@spiffin 

Yes, I used it when my old friend taught me to use CMS to integrate to one of my projects because my clients do not want to write codes. 

I know Textpattern very well and it is easy to code, program, modify and also to install. 

I know you used Base64 to decode the whole `/src/spf_codemirror.php` and this decoded plugins is for being installed on/to Textpattern's plugins.

In spite of it, your plugin is no longer compatible with the latest version of Textpattern, according to Textpattern forum topics. We must implement this plugin, the next months (because I have homeworks and term paper to do, so I am in the last year of computer science course), but I do not know if you will be alive the next months. 

Your instructions do not make sense either because the full stacks, programmers, web developers and web designers can get confused with your instructions and can't figure. We should improve the instructions and write a tutorial tl;dr – install textpattern and set MySQL and `config.php`, after codemirror, after installing your decoded plugin. 

But your plugin is no longer compatible with the latest verison of Textpattern because it is outdated and dated as 2015.

**Comment by gusbemacbe** [comment]
Thank you for having transferred the repository to me. Next months I'll implement and work to improve the bugs as the forum has commented the problem of installation in new version of Textpattern.
